content/docs/getting-started.md

```diff
@@ -100,11 +100,8 @@ Bazen insanlar üçüncü şahıs kitapları ve video kurslarını resmi anlatı
 
 ### Gelişmiş Kavramlar {#advanced-concepts}
 
-<<<<<<< HEAD
+
 [Ana kavramlarda](#main-concepts) bilgi sahibi olup biraz React ile oynadığınızda, daha ileri konulara ilgi duyabilirsiniz. Bu bölüm size [bağlam](/docs/context.html) ve [referanslar](/docs/refs-and-the-dom.html) gibi güçlü fakat daha az kullanılan React özelliklerini tanıtacaktır..
```

Comment:
Since `context` and `refs` are feature names in React, we try to keep them the same as they are as much as possible. So,

> Bu bölüm size [context](/docs/context.html) ve [ref'ler](/docs/refs-and-the-dom.html) gibi ....